src/views/NutritionView.vue

```diff
@@ -297,6 +302,43 @@ export default {
           ];
       }
     },
+    
+    getNutritionData: async function(){
+      let nutritionFromRobotService = await robotoffService.getNutritionValueFromImage(this.productBuffer[0]['code'], this.productBuffer[0]['lang'], this.productBuffer[0]['image_nutrition_url']);
+      this.nutritionData = nutritionFromRobotService.data.nutrients;
+      return this.nutritionData;      
+    },
```

Comment:
Small tip: In JS you have a nice trick to make code easier to read called [destructuring](https://dmitripavlutin.com/javascript-object-destructuring/)

That allows you to write this code ass follow:
```js
// restructuring the attributes I need
const {code, lang, image_nutrition_url} = this.productBuffer[0];

// Reusing them
const nutritionFromRobotService = await robotoffService.getNutritionValueFromImage(code, lang, image_nutrition_url);
```